packages/manager/apps/web/client/app/domain/dashboard/domain.routing.js

```diff
@@ -36,8 +36,8 @@ export default /* @ngInject */ ($stateProvider) => {
       const domainPromise = transition.injector().getAsync('domain');
       return domainPromise.then((domain) =>
         domain.isExpired
-          ? 'app.domain.product.zone'
-          : 'app.domain.product.information',
+          ? 'app.domain.product.information'
+          : 'app.domain.product.zone',
```

Comment:
**question:** Is it the expected behavior? Because I dont think navigating to `Zone` tab if domain is not expired is a good user-experience.